tests/test_bundle.py

```diff
@@ -147,7 +147,7 @@ async def test_deploy_bookinfo_example(ops_test: OpsTest):
         "kubectl",
         "create",
         "namespace",
-        bookinfo_namespace,
+        "bookinfo-namespace",
```

Comment:
why would the namespace be hardcoded here?

Reply:
This is incorrect, leftovers from debugging. Will revert the change. Good catch!

Reply:
Updated.